<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadWindow</class>
 <widget class="QDialog" name="LoadWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>665</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>665</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nuevo Estudio</string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="lblNombreEstudio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>25</height>
    </size>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Nombre del Estudio</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editNombreEstudio">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>545</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>545</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>545</width>
     <height>25</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {border: 0px solid; border-radius: 12; background-color: white; padding: 0 12 0 12}</string>
   </property>
  </widget>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>300</width>
     <height>350</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>350</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="autoScroll">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="indentation">
    <number>10</number>
   </property>
   <property name="headerHidden">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>40</y>
     <width>300</width>
     <height>350</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>350</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>185</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {border: 0px solid; 	border-radius: 12;	background-color: rgb(240,240,240)} QToolButton:hover {border: 0px solid; 	border-radius: 12;	background-color: rgb(178,178,178)}</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
   <property name="arrowType">
    <enum>Qt::RightArrow</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btnRemove">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {border: 0px solid; 	border-radius: 12;	background-color: rgb(240,240,240)} QToolButton:hover {border: 0px solid; 	border-radius: 12;	background-color: rgb(178,178,178)}</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
   <property name="arrowType">
    <enum>Qt::LeftArrow</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>395</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>25</height>
    </size>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {border: 0px solid; 	border-radius: 12;	background-color: rgb(240,240,240)} QToolButton:hover {border: 0px solid; 	border-radius: 12;	background-color: rgb(178,178,178)}</string>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnAgregar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>395</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>25</height>
    </size>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {border: 0px solid; 	border-radius: 12;	background-color: rgb(240,240,240)} QToolButton:hover {border: 0px solid; 	border-radius: 12;	background-color: rgb(178,178,178)}</string>
   </property>
   <property name="text">
    <string>Agregar Estudios</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
